src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Encapsulate an Event which is-a Task.
+ */
+public class Event extends Task {
+
+    protected LocalDate at;
```

Comment:
```suggestion
    protected final LocalDate at;
```
Should this attribute be set as final?

Reply:
I did not implement it as final as I may need to modify the date later on. For example, if a user wants to change the at date in event.